piptools/utils.py

```diff
@@ -525,11 +522,11 @@ def override_defaults_from_config_file(
     ctx: click.Context, param: click.Parameter, value: str | None
 ) -> Path | None:
     """
-    Overrides ``click.Command`` defaults based on specified or discovered config
+    Override ``click.Command`` defaults based on specified or discovered config
     file, returning the ``pathlib.Path`` of that config file if specified or
     discovered.
 
-    ``None`` is returned if no such file is found.
```

Comment:
The original phrase was fine since it's not the title line and is in the middle of the prose / details.